CHANGELOG.md

```diff
@@ -37,6 +37,10 @@ Now they are stored with nanosecond precision on Windows too.
 
 Previously, this would fail with a PKCS#11: CKR_CRYPTOKI_ALREADY_INITIALIZED error.
 
+### fix!: always fetch did file from canister when making canister calls
```

Comment:
This will need to move to the unreleased section after https://github.com/dfinity/sdk/pull/3522 is merged

Reply:
Done